enhancements/installer/ipi-publishing-status.md

```diff
@@ -0,0 +1,221 @@
+---
+title: expose-cluster-publishing-status
+authors:
+  - "@flavianmissi"
+reviewers: # Include a comment about what domain expertise a reviewer is expected to bring and what area of the enhancement you expect them to focus on. For example: - "@networkguru, for networking aspects, please look at IP bootstrapping aspect"
+  - "@patrickdillon, installer"
+  - "@Miciah, edge networking"
+approvers: # A single approver is preferred, the role of the approver is to raise important questions, help ensure the enhancement receives reviews from all applicable areas/SMEs, and determine when consensus is achieved such that the EP can move forward to implementation.  Having multiple approvers makes it difficult to determine who is responsible for the actual approval.
+  - TBD
+api-approvers: # In case of new or modified APIs or API extensions (CRDs, aggregated apiservers, webhooks, finalizers). If there is no API change, use "None"
+  - "@JoelSpeed"
+creation-date: yyyy-mm-dd
+last-updated: yyyy-mm-dd
+tracking-link: # link to the tracking ticket (for example: Jira Feature or Epic ticket) that corresponds to this enhancement
+  - [IR-302](https://issues.redhat.com/browse/IR-302)
```

Comment:
```suggestion
  - https://issues.redhat.com/browse/IR-302
```

This part of the file is YAML embedded in the overall Markdown. That means you can't really use Markdown syntax directly. You can either just have the link, or if you want to try for Markdown you could wrap the value in quotes and see if that makes the YAML parser ignore it.